src/main/kotlin/no/nav/familie/ef/sak/kontantstøtte/KontantstøtteService.kt

```diff
@@ -18,8 +19,22 @@ class KontantstøtteService(
 fun HentUtbetalingsinfoKontantstøtte.tilDto(): HentUtbetalingsinfoKontantstøtteDto =
     HentUtbetalingsinfoKontantstøtteDto(
         this.ksSakPerioder.isNotEmpty() || this.infotrygdPerioder.isNotEmpty(),
+        this.ksSakPerioder.map { KsakPeriodeDto(it.fomMåned, it.tomMåned, KontantstøtteDatakilde.KONTANTSTØTTE) } +
+            this.infotrygdPerioder.map { KsakPeriodeDto(it.fomMåned, it.tomMåned, KontantstøtteDatakilde.INFOTRYGD) },
     )
 
 data class HentUtbetalingsinfoKontantstøtteDto(
     val finnesUtbetaling: Boolean,
+    val perioder: List<KsakPeriodeDto>,
 )
+
+data class KsakPeriodeDto(
```

Comment:
```suggestion
data class KontantstøttePeriodeDto(
```

Reply:
Her er vel poenget å skille mellom infotrygd, kssak og ikke minst "våre interne" kontantstøtteperioder, så KsSakPeriode er forsåvidt greit, men da `KsSakPeriodeDto` (mangler en stor S)

Reply:
KsSak = kontantstøtteSak? Kunne man isåfall presisert dette?